### DISCUSSION TITLE
Check the custom domain DNS records regularly instead of manual check by user

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#549] [Feature Requests]
Currently, a person has to click on "verify" to manually check a domain record. It'd be better to have a background task to check the DNS records and notify this person when the DNS is correctly set up.

**Comment by jumpingbrowntrout** [comment]
Hope to see this one get some upvotes.

Supporting this feature will allow for quicker cutovers and less downtime/lost emails whilst MX server's are being migrated.
With the current method, the domain doesn't come active within simplelogin until the user manually verifies it. There are two problems with this:

1) There DNS requests are limited and possibly cached so it can take hours, in my case up to 4hrs before the DNS request is successful with the MX servers pointing to simplelogin. Any emails sent to you in this waiting period are rejected by the mail server and therefore lost forever.
2) Since you need to manually verify if it's late night and you need to sleep or be offline your email could be down/rejected until you can next try re-verify.
